packages/server/tests/acceptance/ws/sendRawTransaction.spec.ts

```diff
@@ -0,0 +1,115 @@
+/*-
+ *
+ * Hedera JSON RPC Relay
+ *
+ * Copyright (C) 2023-2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+// external resources
+import { expect } from 'chai';
+import { ethers, WebSocketProvider } from 'ethers';
+import RelayClient from '../../clients/relayClient';
+import MirrorClient from '../../clients/mirrorClient';
+import { AliasAccount } from '../../clients/servicesClient';
+import { numberTo0x } from '@hashgraph/json-rpc-relay/src/formatters';
+import { ONE_TINYBAR_IN_WEI_HEX } from '@hashgraph/json-rpc-relay/tests/lib/eth/eth-config';
+
+describe('@web-socket eth_sendRawTransaction', async function () {
```

Comment:
Q: Will this be run with acceptance tests on release qualifying runs?

Reply:
It looks like these `@web-socket` tests are not run on releases. Here is the [CI command](https://github.com/hashgraph/hedera-json-rpc-relay/blob/main/.github/workflows/release-acceptance.yml#L65) in release CI, it runs the `acceptancetest:release`  and this command only runs tests with `@release` tag like bellow.

```
"acceptancetest:release": "ts-mocha packages/server/tests/acceptance/index.spec.ts  -g '@release' --exit",
```

Eric suggested we should include `@release` tags to all methods that ChainLink requires. So I just pushed in an update for it.